pack/fhv/fhvc.json

```diff
@@ -118,9 +118,419 @@
+    "illustrator": "Rob Laskey",
+    "name": "A Botanical Survey",
+    "pack_code": "fhvc",
+    "position": 91,
+    "quantity": 1,
+    "stage": 1,
+    "text": "<b>Objective</b> – At the end of the round, investigators at the North Shore location may spend the requisite number of clues, as a group, to advance. <i>(See the Location Placement diagram on page 28 of the campaign guide for the North Short location.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_name": "Strange Anatomy",
+    "back_text": "<blockquote><i>You corner the reptilian creature in a shining pool, where it sinks into the mud, clearly exhausted. As it starts munching on a tangle of flowers, you take several sketches and notes of the thing's sprouting shell. The alkaloid blossom on its back seems to pulse. Producing a pair of shears, you prepare to take a sample.</i></blockquote>\n<hr>If it is <b>Day 1</b>, read the following. Otherwise, <b>(→R1)</b>.\n<blockquote><i>\"Stop!\" Dr. Marquez shouts. \"That flower is parasitic. It is now part of the creature's anatomy. There's no telling how deeply it's embedded.\" She points to the flower's thick translucent stem: vicious white fluid pumps through it like an artery. \"If the flower dies, its host dies too.\"</i></blockquote>\n<b>(→R2)</b> <i>(page 30)</i>.",
+    "code": "10592",
+    "clues": null,
+    "double_sided": true,
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "flavor": "The large flowering turtle-like creature is on the move. Perhaps you can herd it to safety.",
+    "illustrator": "Rob Laskey",
+    "name": "Discovery of a Lifetime",
+    "pack_code": "fhvc",
+    "position": 92,
+    "quantity": 1,
+    "stage": 2,
+    "text": "[fast] Investigators at a single location adjacent to Chelydran Hybrid's location spend 1 [per_investigator] clues, as a group: For the remainder of the round, change Chelydran Hybrid's patrol location to that location.\n<b>Forced</b> – At the end of the enemy phase: Chelydran Hybrid takes 1 damage for each ready [[Abomination]] enemy at its location.\n<b>Objective</b> – At the end of the round, if the Chelydran Hybrid is at the Starting Location, immediately advance.",
+    "type_code": "act"
+  },
+  {
+    "back_link": "10593b",
+    "clues": 2,
+    "code": "10593a",
+    "encounter_code": "the_thing_in_the_depths",
+    "encounter_position": 6,
+    "faction_code": "mythos",
+    "flavor": "For some unknown reason I dreaded the thought of disturbing the ancient bog and its sunless secrets.\n – H. P. Lovecraft, \"The Moon-Bog\"",
+    "illustrator": "Pixoloid Studios",
+    "name": "Muddy Fen",
+    "pack_code": "fhvc",
+    "position": 93,
+    "quantity": 2,
+    "shroud": 4,
+    "traits": "Bog.",
+    "type_code": "location",
+    "victory": 1
+  },
+  {
+    "code": "10593b", 
```

Comment:
This is missing `clues: null`:
<img width="909" alt="Screenshot 2024-10-22 at 13 38 21" src="https://github.com/user-attachments/assets/97046d87-a3d5-4273-9ace-cb36a20e1f15">


```suggestion
    "code": "10593b",
    "clues": null,
```